src/analysis/bounded_variation.lean

```diff
@@ -207,6 +207,56 @@ begin
   simp [u, edist_comm],
 end
 
+lemma lower_continuous_aux {ι : Type*} {F : ι → α → E} {p : filter ι}
```

Comment:
The statement of the lemma is not the right one: you don't need uniform convergence for that, pointwise convergence is enough. Better write a lemma assuming convergence at every point of `s`, and then deduce one for uniform convergence.

Reply:
Huh, you're right, thanks!